testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -594,6 +595,63 @@ def test_sequence_alignment_deprecation(self, atomgroups):
             align.sequence_alignment(mobile, reference)
 
 
+class TestIterativeAverage(object):
+    @pytest.fixture()
+    def mobile(self):
+        u = mda.Universe(PSF, DCD)
+        return u
+
+    @pytest.fixture()
+    def reference(self):
+        u = mda.Universe(PSF, DCD)
+        return u
+
+    def test_iterative_average(self, mobile, reference):
+        res = align.iterative_average(mobile, niter=1)
+        res = align.iterative_average(mobile, reference, niter=1, eps=0)
+        res = align.iterative_average(mobile, reference, eps=1e-5)
+        res = align.iterative_average(mobile, reference, select='bynum 1:10',
+                                      weights=np.ones(10))
+
+        res = align.iterative_average(mobile, reference, select='bynum 1:10',
+                                      niter=10, verbose=True)
+        assert_almost_equal(
```

Comment:
just use assert_allclose (or is there a specific reason for   assert_almost_equal ??)